lib/tasks/migrate_results_to_accountability.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+namespace :migrate do
+  desc "Copy results from Decidim::Results feature to Decidim::Accountability feature for process (default 1 - PAM)"
+  task :results_to_accountability, [:process_id] => :environment do |_, args|
```

Comment:
This can be done by adding all `.rake` files as an exception here: https://github.com/AjuntamentdeBarcelona/decidim-barcelona/blob/master/.rubocop.yml#L1276